src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -1322,6 +1457,22 @@ impl GraphEditorModel {
         self.breadcrumbs.set_position_y(y_offset);
         self.breadcrumbs.gap_width(traffic_lights_gap_width());
         self.scene().add_child(&self.tooltip);
+
+        self.add_child(&self.profiling_button);
+        self.scene().layers.breadcrumbs_background.add_exclusive(&self.profiling_button);
+        self.profiling_button.set_visibility(true);
+        self.profiling_button.set_size(Vector2(32.0,32.0));
+
+        let styles = StyleWatch::new(&self.scene().style_sheet);
+        let color_scheme = toggle_button::ColorScheme {
+            non_toggled     : Some(styles.get_color(theme::graph_editor::profiling_button::non_toggled).into()),
+            toggled         : Some(styles.get_color(theme::graph_editor::profiling_button::toggled).into()),
+            hovered         : Some(styles.get_color(theme::graph_editor::profiling_button::hovered).into()),
+            toggled_hovered : Some(styles.get_color(theme::graph_editor::profiling_button::toggled_hovered).into()),
+            ..default()
+        };
+        self.profiling_button.set_color_scheme(&color_scheme);
```

Comment:
The initialization of the profiling button should be refactored to a module containing it to prevent this file from growing and to partition resposnibilities.